src/components/overlays/Tooltip/TooltipProvider.stories.tsx

```diff
@@ -132,6 +134,23 @@ export const TooltipWithScroll: Story = {
   ),
 };
 
+
+export const TooltipInCardHeader: Story = {
+  render: () => (
+    <Card >
+      <Card.Heading>
+        TooltipInCardHeader
+        <TooltipProvider tooltip="Tooltips will auto-reposition when it hits the viewport due to scroll.
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. ThisIsAVeryLongStringOfTextWithoutAnySpacesToTestWhetherWeHandleWordBreaksCorrectlyWhenTheTextOverflowsTheContainingElement.
+
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. ThisIsAVeryLongStringOfTe"
+        >
+          <Icon icon='info' />
+        </TooltipProvider>
+      </Card.Heading>
+    </Card>
+  )
+}
```

Comment:
- Add a trailing comma after the `)`
- Add a semicolon after the `}`
- Add a newline to separate this story from the next